lib/mayaUsd/render/vp2RenderDelegate/basisCurves.cpp

```diff
@@ -1193,12 +1199,13 @@ HdVP2BasisCurves::_UpdateDrawItem(
     }
 
     // Determine if the render item should be enabled or not.
-    if (itemDirtyBits & (HdChangeTracker::DirtyVisibility |
+    if ((itemDirtyBits & (HdChangeTracker::DirtyVisibility |
                          HdChangeTracker::DirtyRenderTag |
                          HdChangeTracker::DirtyPoints |
                          HdChangeTracker::DirtyExtent |
-                         DirtySelectionHighlight)) {
-        bool enable = drawItem->GetVisible() && !_curvesSharedData._points.empty();
+                         DirtySelectionHighlight)) ||
+                         !GetInstancerId().IsEmpty()) {
```

Comment:
The old expression checking instancerWithNoInstances wouldn't re-show the render item when going from 0 instances to 1 or more instances.

Reply:
Do you find a way to interactively add instances?

Reply:
I think the python API may be able to do it, but I didn't actually get it to work. I just realized it was a problem when updating the code.